Snowflake.Data/Snowflake.Data.csproj

```diff
@@ -1,7 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>net6.0;net471;net472</TargetFrameworks>
-    <TargetFrameworks Condition="'$(OS)' != 'Windows_NT'">net6.0</TargetFrameworks>
+    <TargetFramework>netstandard2.0</TargetFramework>

```

Comment:
we have in this file: `<ItemGroup Condition="'$(TargetFramework)' == 'net471' Or '$(TargetFramework)' == 'net472'">` Shouldn't we modify it?

Reply:
It can be removed since only netstandard2.0 is targeted

Reply:
Removed

Reply:
We have generally found it best to target both netstandard2.0 and net6.0,  net8.0 etc. I think it then means that all the extra dlls that get included when bringing in netstandard packages don't get included in the output as they're all included with the runtimes.

I might be out of date though